src/ClaverTapProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Flits\ClavertapSdk;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7;
+use Flits\ClavertapSdk\ClaverTapException;
+
+class ClaverTapProvider {
+    public $BASE_URL = "https://in1.api.clevertap.com/1/upload";
```

Comment:
The base URL should be base.

Currently, it is for the upload API.

Upload API URL should be in the API/upload file.